src/mmw/apps/modeling/views.py

```diff
@@ -141,22 +141,10 @@ def project_weather(request, proj_id, category):
         return Response({'errors': ['Invalid category specified.']},
                         status=status.HTTP_400_BAD_REQUEST)
 
-    if category == 'NASA_NLDAS_2000_2019' \
-            and not (project.in_drwi or project.in_pa):
-        return Response({'errors': ['Only supported within'
-                                    ' Delware River Watershed Initiative'
-                                    ' or the Commonwealth of Pennsylvania.']},
-                        status=status.HTTP_400_BAD_REQUEST)
-
-    if category in ['RCP45_2080_2099', 'RCP85_2080_2099'] \
-            and not project.in_drb:
-        return Response({'errors': ['Only supported within'
-                                    ' Delware River Basin.']},
-                        status=status.HTTP_400_BAD_REQUEST)
-
     # Get weather simulation data for this project
     mods, errs = get_weather_simulation_for_project(project, category)
 
+    # TODO We should handle 404s here
```

Comment:
Currently `get_weather_simulation_for_project` returns a list of text errors. Will have to devise a way to extract the status code from that, and return a 400 or 404 as appropriate.